packages/web/src/pages/sign-up-page/components/SocialMediaLoading.tsx

```diff
@@ -0,0 +1,32 @@
+import { Box, Flex, Text } from '@audius/harmony'
+
+import LoadingSpinner from 'components/loading-spinner/LoadingSpinner'
+
+import styles from './SocialMediaLoading.module.css'
+
+const messages = {
+  hangTight: 'Hang tight!',
+  connectingSocialMedia: 'We’re connecting your social account'
+}
+
+export const SocialMediaLoading = () => {
+  return (
+    <Flex
+      direction='column'
+      alignItems='center'
+      h='100%'
+      justifyContent='center'
+      gap='2xl'
+      css={{ textAlign: 'center', marginBottom: '20%' }}
+    >
+      <Text variant='heading' size='l' color='accent'>
+        {messages.hangTight}
+        <br />
+        {messages.connectingSocialMedia}&hellip;
+      </Text>
+      <Box mb='2xl'>
+        <LoadingSpinner className={styles.spinner} />
```

Comment:
css prop can be `css={{g: {path: {stroke: theme.color.neutral['n-800']}}` (i forget the exact theme variable)

Reply:
Can't put the css prop on this since it comes from web :/